grammar.js

```diff
@@ -171,7 +171,7 @@ module.exports = grammar({
     ),
 
     // Modeline: must start with "vim:" (optionally preceded by whitespace)
-    modeline: ($) => token(prec(2, /[\t ]*vim:[^\n]+\n/)),
+    modeline: ($) => token(prec(2, /vim:[^\n]+\n/)),
```

Comment:
Yeah, including explicit whitespace in the token always causes problems. And it's not needed here anyway, because `modeline` is defined as a top-level ("block") token: https://github.com/neovim/tree-sitter-vimdoc/blob/66f4fee37b07cff0dd3400089eebccd02e61e5c2/grammar.js#L25-L29

Reply:
Yup. This slipped by me in review.